spec/lob/resources/postcard_spec.rb

```diff
@@ -37,12 +37,38 @@
         description: @sample_postcard_params[:description],
         to: new_address["id"],
         message: @sample_postcard_params[:message],
-        front: "https://lob.com/postcardfront.pdf",
+        front: "https://lob.com/postcardfront.pdf"
       )
 
       result["description"].must_equal(@sample_postcard_params[:description])
     end
 
+    it "should accept an idempotency_key" do
```

Comment:
A pattern that I've used before that might work is to put this test in the `resource_test.rb` file, and in that file, create a "test" resource that mimics a real one. 

So within the test file, creating a class like:

``` ruby
module Lob
  module Resources
    class TestResource < Lob::Resources::ResourceBase

    # ...

    end
  end
end
```

We've used this pattern in Dora sometimes. What do you guys think?